frontend/app/common/fetcher.ts

```diff
@@ -104,36 +77,33 @@ const fetcher = methods.reduce<Partial<FetcherObject>>((acc, method) => {
 
             throw new RequestError(descriptor.defaultMessage, res.status);
           }
+
           return res.text().then((text) => {
             let err;
             try {
               err = JSON.parse(text);
             } catch (e) {
-              if (logError) {
-                console.error(err);
-              }
-
               throw new RequestError(httpMessages.unexpectedError.defaultMessage, 0);
             }
             throw err;
           });
         }
 
-        if (res.headers.get('Content-Type')?.startsWith('application/json')) {
+        if (res.headers.get('Content-Type')?.indexOf('application/json') === 0) {
           return res.json();
         }
 
         return res.text();
-      })
-      .catch((e) => {
-        if (isFailedFetch(e)) {
+      } catch (e) {
+        if (e?.message === 'Failed to fetch') {
           throw new RequestError(e.message, -2);
         }
 
         throw e;
-      });
-  };
-  return acc;
-}, {}) as FetcherObject;
+      }
+    };
+    return acc;
+  }, {} as FetcherMethods);
 
-export default fetcher;
+export const apiFetcher = createFetcher(`${BASE_URL}${API_BASE}`);
```

Comment:
For consistency, if you are creating fetcher a level higher in one place, let's create apiFetcher a level higher as well?

Reply:
`apiFetcher` now is used in two places. I would like to create it in other place when I'll find out how to tree shake all unused functions from file if one of our bundles used only one method from it.

Reply:
Let's move `authFetcher` and `adminFetcher` in and don't export `createFetcher` then? It doesn't seem right to have two approaches both in the code at the same time.